### PR TITLE
feat: implemented lerna

### LINKED ISSUE #204: Introduce monorepo with lerna
We want to have documentation and ui framework itself in different packages.

## Why
While we technically can jam 2 things in one project, build requirements are too diverse:
* for documentation we want nuxt build (for SEO), markdown support etc.
* for lib it's likely going to be vue-cli for demos + rollup for compiling.

## Things to do
- [x] init lerna
- [x] move current docs to `docs` lerna package
- [x] move other files to `ui` lerna package.
- [x] ensure key scripts (`serve`, `build`, `test`, `lint`) work at project root level.

## Refs
* [vuetify](https://github.com/vuetifyjs/vuetify) - they have pretty much the same thing, lerna structure wise.

### LINKED ISSUE #205: Introduce nuxt build for documentation
The purpose for this task is to provide boilerplate for future building of doc engine.
Currently vuepress is being used and it's not flexible enough (we want to use framework components to build docs).

Blocked by https://github.com/epicmaxco/vuestic-ui/issues/204

## Why nuxt
* Nuxt allows us to get SEO without too much hassle.
* We actually want to test our components against nuxt.
* Docs on nuxt are good way to showcase epicmax experience.

## Details
- [x] move existing documentation to `packages/old-docs` folder.
- [x] create nuxt project in `packages/docs`, add TypeScript support.
- [ ] Configure CI to run nuxt in static build (ping @asvae if unclear).

### PR DESCRIPTION
**Pull request by SergeiPleshkov**
Closes #204 
Closes #205 

## Description

Implemented monorepo using lerna. Implemented nuxt builerplate for new docs package, change ci config.

## Details:

<details>
Project sructure now is:
root
--packages
----old-docs
----docs
----ui
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@vuestic-ui/docs",
+  "version": "0.1.8",
+  "description": "Documentation for vuestic-ui framework",
+  "main": "index.js",
+  "repository": "https://github.com/epicmaxco/vuestic-ui",
+  "license": "MIT",
+  "dependencies": {
+    "core-js": "2",
```
Comment (asvae) [inline packages/docs/package.json:9]: Just 2? o.o

File: packages/docs/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (asvae) [inline packages/docs/tsconfig.json:1]: Might make sense to double check these from dev.

File: packages/ui/package.json [hunk 3]
```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@vuestic-ui/ui",
```
Comment (asvae) [inline packages/ui/package.json:2]: should be `vuestic-ui`. Name corresponds to npm package.

File: packages/ui/package.json [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  "version": "1.0.0-alpha.2",
+  "license": "MIT",
+  "repository": "https://github.com/epicmaxco/vuestic-ui",
+  "scripts": {
+    "serve": "vue-cli-service serve --mode book",
```
Comment (asvae) [inline packages/ui/package.json:7]: Problem with serve is that there's nothing in console... So it's not useful.

File: packages/docs/package.json (hunk 1 above)
Comment (SergeiPleshkov) [reply packages/docs/package.json:9]: It is recommended in official docs to use v2, not 3. 3 causes module errors.

**Pull request merged**